kdd-2024/ibis-tutorial-KDD2024.md

```diff
@@ -0,0 +1,286 @@
+# Descriptive Title 
+
+- "Ibis: The Data Scientist's Powerhouse for Blazing-Fast Analytics and Machine Learning"
```

Comment:
nit: Powerhouse and Blazing-fast are both buzzwords, I'd suggest giving a more direct title on why Ibis is the powerhouse and why it's blazing-fast on large scale data

Reply:
Sure, Do you have a suggestion? We were trying to get a title that targets DS and ML.

Reply:
how about something like: "Unified Machine Learning & Analytics workflows with Ibis: bridging batch and streaming from local notebook to production scale"

Reply:
I would change "Powerhouse" to "Toolkit" -- I don't think the focus of this talk as-is is on bridging batch to local though. we should probably focus on our strengths today (Analytics) and the next-level things (ML, Streaming)

perhaps:

"Ibis: The Data Scientist's Toolkit for Blazing-Fast Analytics, Machine Learning, and Stream Processing"

Reply:
boom.  :shipit: